test/Chirp.Infrastructure.Tests/RestrictedCheepsUnitTests.cs

```diff
@@ -32,7 +32,7 @@ public RestrictedCheepTests()
         };
         var testCheep = new Cheep
         {
-            CheepId = 1,
+            CheepId = new Guid(1, 0, 0, new byte[]{0,0,0,0,0,0,0,0}),
```

Comment:
I think that this is not the way you use Guid. You should use Guid.NewGuid() instead of new Guid().